### PR TITLE
#19 Maya v1.0.0 Tools docs

### PR DESCRIPTION
**Pull request by lucasferryinbibo**
# Description of this issue
Tools documented.

# Pages to review
tools.md

Issue of this PR: https://github.com/Inbibo/adonisfx_docs/issues/19

### CODE REVIEW

File: docs/documentation/maya/tools.md [hunk 1]
```diff
@@ -1,168 +1,186 @@
...
 
-In the specific case of adnMuscle deformers, the AdonisFX Paint Tool will display the following attributes:
+In the specific case of AdnMuscle deformers, the AdonisFX Paint Tool will display the following attributes:
 
-<figure markdown> 
+<!-- Note for later to update this information once the Global Damping attribute is added-->
+<figure markdown>
   ![Pain Tool Skin Attributes example](images/paint_tool_ribbon_attributes.png) 
-  <figcaption> Figure 3: AdonisFX paint tool displaying the paintable attributes of the deformer. </figcaption> 
+  <figcaption> Figure 6: AdonisFX paint tool displaying the paintable attributes of the deformer. </figcaption>
```
Comment (lucasferryinbibo) [inline docs/documentation/maya/tools.md:94]: Update screenshot with Global Damping showing in the list after update

File: docs/documentation/maya/tools.md [hunk 2]
```diff
@@ -1,168 +1,186 @@
...
 
-In the specific case of adnSkin deformers, the AdonisFX Paint Tool will display the following attributes:
+#### AdonisFX Paint Tool on AdnSkin deformers
+
+In the specific case of an AdnSkin deformer, the AdonisFX Paint Tool will display the following attributes:
 
 <figure markdown> 
   ![Pain Tool Skin Attributes example](images/paint_tool_skin_attributes.png) 
-  <figcaption> Figure 4: Paintable attributes listed in the UI for adnSkin deformers. </figcaption>
+  <figcaption> Figure 8: Paintable attributes listed in the UI for an AdnSkin deformer. </figcaption>
```
Comment (lucasferryinbibo) [inline docs/documentation/maya/tools.md:137]: Update screenshot with Global Damping showing in the list after update

**Review by lucasferryinbibo — COMMENTED** [review]
I've left a comment to update some screenshots

File: docs/documentation/maya/tools.md [hunk 3]
```diff
@@ -1,168 +1,186 @@
...
 
 ## AdonisFX Import Tool
 
-The **AdonisFX Import Tool** allows the user to import the configuration for a specific defomrer from a previously exported AAD (Adonis Attribute Definition) file.
+The **AdonisFX Import Tool** allows the user to import the configuration for a specific deformer from a previously exported AAD (Adonis Asset Definition) file. It allows the user to easily configure a deformer from an already known setup which includes data like weight maps, connections and attribute values.
 
 <figure>
   <img src="images/import_tool_empty.png" caption="AdonisFX Import Tool">
-  <figcaption>Figure 1: AdonisFX Import Tool</figcaption>
+  <figcaption>Figure 1: AdonisFX Import Tool UI</figcaption>
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/tools.md:11]: Add bold style to all "Figure x":
```suggestion
  <figcaption><b>Figure 1</b>: AdonisFX Import Tool UI</figcaption>
```

File: docs/documentation/maya/tools.md [hunk 4]
```diff
@@ -1,168 +1,186 @@
...
+To open the tool press the ![Import Tool](images/adn_exporter.png) shelf button or go to AdonisFX Menu > *Export*.
 
-Once that is set we must select the *Output File Path*, to which the AAD file will get exported.
+To export and apply an ADD file for the current entity in the current open scene with the deformer information follow these steps:
 
-Once everything is properly set up, pressing the *Export* button will generate the file in the designated location.
+1. If the tool had been opened on an already active selection containing an Adonis deformer the node will appear in the "Select Export Data" list. To add and remove a new selection press *Add Selection* or *Clear*.
+2. Select, using the checkboxes, the deformers and the individual settings to export to the AAD file. For example selecting "Maps" and "Attachments" would export the painted attribute maps and the matrices for the attachments relative to an AdnMuscle or AdnRibbonMuscle deformer.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/tools.md:55]:
```suggestion
2. Select, using the checkboxes, the deformers and the individual settings to export to the AAD file. For example selecting *Maps* and *Attachments* would export the painted attribute maps and the matrices for the attachments relative to an AdnMuscle or AdnRibbonMuscle deformer.
```

**Review by josevalverdeinbibo — CHANGES_REQUESTED** [review]
Great job! Some comments to check.

File: docs/documentation/maya/tools.md (hunk 3 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/tools.md:11]: Done.

File: docs/documentation/maya/tools.md [hunk 5]
```diff
@@ -1,168 +1,186 @@
...
+1. Locate the ADD file by going to *Input File Path's* folder button and select an AAD file from a directory.
+2. Locate the available deformer names that were fetched from the AAD file on the left section of the tool ("Node" | "Node Type").
+3. Locate the available nodes found in the current scene selection to which the AAD file configuration can be applied to ("Mesh" | "Candidate Node"). The selection can be redone and expanded by selecting elements from the scene and pressing the *Add Selected* button or can be cleared using the *Clear* button.
+4. Select the deformer from which the AAD file data will be fetched in the left hand panel.
+5. Select the node to which to apply the selected deformer configuration in the right hand panel.
+6. Press the *Assign* button to temporarily assign the configuration to the target node. Or clear the assignment using *Delete*.
+7. The assignments can then be imported using the **Import* button or everything can be cleared using the *Clear All* button.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/tools.md:26]: Done.

File: docs/documentation/maya/tools.md (hunk 4 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/tools.md:55]: Done.

File: docs/documentation/maya/tools.md [hunk 6]
```diff
@@ -1,168 +1,186 @@
...
+Two main sections can be distinguished in this tool, labeled *source* and *destination*. In the source section the [sensors](sensors.md), alongside their signal attributes, will get displayed and in the destination section the **deformers**, alongside their possible input attributes, will get displayed.
 
-To retrieve these objects and display them in the tool, select the desired element from the scene (an Adonis locator containing a sensor or a deformer) and press their respective **Reload Left** or **Reload Right** button.
+To retrieve these objects and display them in the tool, select the desired element from the scene (an Adonis locator containing a sensor or a deformer) and press their respective *Reload Left* or *Reload Right* button.
 
-For source elements (sensors) press the **Reload Left** button and for destination elements (deformers) press the **Reload Right** button.
+For Source elements (sensors) press the *Reload Left** button and for Destination elements (deformers) press the *Reload Right* button.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/tools.md:177]: Done.

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Pull request merged**